packages/tpetra/core/src/Tpetra_Import_Util2.hpp

```diff
@@ -712,6 +719,19 @@ sortAndMergeCrsEntries (const Teuchos::ArrayView<size_t> &CRS_rowptr,
   return sortAndMergeCrsEntries(CRS_rowptr, CRS_colind, CRS_vals);
```

Comment:
I am not quite sure, but this can be updated eventually, at this point it seems to be a host only view based function.
Not too sure who wanted it that way, we can make an issue about that and fix it after this PR gets merged.